src/main/java/hudson/plugins/analysis/collector/AnalysisPublisher.java

```diff
@@ -149,6 +68,21 @@ public boolean isCheckStyleActivated() {
     }
 
     /**
+     * Needed by workflow snippet generator.
+     * Having this ugly getter naming to keep backward configuration file compatibility on fields names.
+     * 
+     * @see #isCheckStyleActivated()
+     */
+    public boolean isIsCheckStyleActivated() {
```

Comment:
Sorry, now I see that the misnamed `@DataBoundSetter` was newly introduced. So just

1. delete the new getter
1. fix the name of the new setter
1. fix the misnamed properties in `tools.jelly`

BTW if you are fixing Jelly, you can probably simplify

```xml
<f:entry title="${%Checkstyle warnings}">
    <f:checkbox name="isCheckStyleActivated" checked="${h.defaultToTrue(instance.isCheckStyleActivated())}"/>
</f:entry>
```

to

```xml
<f:entry field="isCheckStyleActivated" title="${%Checkstyle warnings}">
    <f:checkbox default="true"/>
</f:entry>
```

before renaming the `field`.